tests/Model/Generated/Endpoint/SessionTest.php

```diff
@@ -45,4 +35,14 @@ public static function tearDownAfterClass()
         BunqContext::getApiContext()->resetSession();
         BunqContext::getApiContext()->save(self::FILE_PATH_CONTEXT_CONFIG);
     }
+
+    /**
+     * Delete's the current session.
+     *
+     * This test has no assertion as of its testing to see if the code runs without errors.
```

Comment:
`as it is testing whether the code runs without errors.`

Same typo as in other PR. Generated/copied error?

Reply:
Nah, back in the day when we were using the old PhpUnit you could leave tests without assertions hence the message in the test header. Since #96 it will say risky tests if no assertions where added. But I seem to forgot to remove these comments 🤦‍♂️